Makefile

```diff
@@ -1,136 +1,102 @@
 TERRAFILE_VERSION=0.8
 ARM_TEMPLATE_TAG=1.1.6
 RG_TAGS={"Product" : "Get Schools Experience"}
-SERVICE_SHORT=gse
 REGION=UK South
-
```

Comment:
The original Makefile commands must be kept as they are in use by the live service